src/main/webapp/data-visualization.js

```diff
@@ -33,11 +33,11 @@ async function displayVisualization(censusDataArray, description,
   setStyle(isCountyQuery);
   amChartsData = createDataArray(censusDataArray, isCountyQuery);
   globalDescription = description;
-  drawTable(amChartsData, isCountyQuery);
+  drawTable(amChartsData, globalDescription, isCountyQuery);
   if (isCountyQuery) {
       const mapsData = getMapsData(censusDataArray);
       displayAmChartsMap(amChartsData, globalDescription, globalGeoData, '#3c5bdc');
-      displayCountyGeoJson(mapsData, location);
+      displayCountyGeoJson(mapsData, globalDescription, location);
   } else {
       displayAmChartsMap(amChartsData, description, globalGeoData, '#3c5bdc');
```

Comment:
'description' could be 'globalDescription' on this line for consistency throughout the function

Reply:
Thanks! Fixed